superset-frontend/stylesheets/less/cosmo/variables.less

```diff
@@ -91,7 +91,7 @@
 //
 // ## Define common padding and border radius sizes and more. Values based on 14px text and 1.428 line-height (~20px to start).
 
-@padding-base-vertical: 10px;
+@padding-base-vertical: 6.5px;
```

Comment:
Since this is part of the broader bootstrap theme, I'm a little worried this might cause some other regression somewhere else in the application. Have you done some kind of visual audit, or could we make this change in the relevant component(s) with Emotion styling?

Reply:
I did a visual audit of most screens and I didn't found any problem. It shouldn't cause any problem because this is a configurable variable of Bootstrap. To limit the scope of this change we need to track all components that use classes that depend on this variable and apply Emotion styles individually.